container_files/assets/www/doc/builtin/functions/Stemmer.md

```diff
@@ -36,8 +36,33 @@ PUT /v1/functions/my_stemmer
 }
 ```
 
-we can tokenize a sentence and apply the stemming algorithm on the
-resulting tokens:
+and this dataset:
+
+| *rowName* | *potato* | *potatoes* | *carrot* | *carrots* |
+|-----------|----------|------------|----------|-----------|
+|   row_0   |     1    |      2     |     3    |    None   |
+|   row_1   | 'crips'  |  'chips'   |     0    | 'hi mom'  |
+
+
+The following query will merge the stemmed columns, summing their content:
+
+```
+SELECT my_stemmer({words:{*}})[words] as * FROM our_dataset
+```
+
+and return:
+
+
+| *rowName* | *potato* | *carrot* |
+|-----------|----------|----------|
+|   row_0   |     3    |     3    |
+|   row_1   |     2    |     1    |
+
+
+Note that strings are coerced to the integer value 1.
+
+
+We can also nicely use it in conjunction with the tokenize function:
 
 ```
 SELECT my_stemmer({words: {tokenize('I like having lots', {splitchars:' '}) as *}}) as *
```

Comment:
I'll use a phrasing that doesn't add another word though, like "I have liked having lots".